pkg/operator/ceph/csi/spec.go

```diff
@@ -272,7 +271,10 @@ func StartCSIDrivers(namespace string, clientset kubernetes.Interface, ver *vers
 		logger.Warning("CSI Block volume expansion requires Kubernetes version >=1.16.0")
 	}
 
-	logLevel := os.Getenv("CSI_LOG_LEVEL")
+	logLevel, err := k8sutil.GetOperatorSetting(clientset, "CSI_LOG_LEVEL", "")
+	if err != nil {
+		logger.Errorf("failed to load CSI_LOG_LEVEL. Defaulting to %d. %v", defaultLogLevel, err)
```

Comment:
Typically we would return when there is an error. Since we're not returning, please add a comment stating that we intentionally continue

Reply:
When the setting fails to load and we fall back to a default, how about we log a warning instead of an error? If it's an error we should really treat it as serious and return.

Reply:
Will change.